1/tom-stuart-captions.srt

```diff
@@ -3213,7 +3213,7 @@ I think that's fabulous.
 693
 00:42:24,800 --> 00:42:27,080
 TS: I think it's just
-called a 'git', git gutter.
+called 'git gutter'.
```

Comment:
Tom quietly does correct himself to "vim-gitgutter" (the actual name of the plugin) but as it is visible on screen, I thought we could cut this down to his most assertive "git gutter".